modules/util/create.py

```diff
@@ -336,12 +372,15 @@ def create_optimizer(
                 nesterov=optimizer_config.nesterov if optimizer_config.nesterov is not None else False,
                 foreach=optimizer_config.foreach if optimizer_config.foreach is not None else False,
                 maximize=optimizer_config.maximize if optimizer_config.maximize is not None else False,
-                differentiable=optimizer_config.differentiable if optimizer_config.differentiable is not None else False,
+                differentiable=optimizer_config.differentiable
```

Comment:
This might be a limitation of the formatter, but here I think the if and else line should be indented one more level. All three lines belong to the same statement. Indenting them all the same way makes it impossible to see that they are not the same. This same thing happened multiple times in this file

Reply:
Disabled formatter.